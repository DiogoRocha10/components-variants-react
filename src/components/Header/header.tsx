import styles from "./header.module.css"

interface HeaderProps {
	label: string
}

export const Header = ({ label }: HeaderProps) => (
	<header>
		<div className={styles.wrapper}>
			<div>
				<h1>{label}</h1>
			</div>
		</div>
	</header>
)
