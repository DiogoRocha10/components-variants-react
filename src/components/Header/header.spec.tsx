import { render } from "@testing-library/react"
import { Header } from "./header"
import renderer from "react-test-renderer"

it("text", () => {
	const { getByText } = render(<Header label="Header" />)
	expect(getByText("Header")).toBeTruthy()
})

it("renders correctly", () => {
	const tree = renderer.create(<Header label="Header" />).toJSON()
	expect(tree).toMatchSnapshot()
})
