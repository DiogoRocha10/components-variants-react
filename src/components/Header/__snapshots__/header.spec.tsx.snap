// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <div
    className="wrapper"
  >
    <div>
      <h1>
        Header
      </h1>
    </div>
  </div>
</header>
`;
